scss/_alerts.scss

```diff
@@ -66,77 +66,77 @@
       }
 
       p {
-        color: inherit;
         font-size: $font-size-sm;
+        color: inherit;
       }
     }
   }
 }
 
 .alert-info {
-  border-color: theme-color('info');
+  border-color: theme-color("info");
 
   &::before {
-    content: '\E88F';
-    background-color: theme-color-background('info');
-    color: theme-color('info')
+    color: theme-color("info");
+    content: "\E88F";
+    background-color: theme-color-background("info");
   }
 
   .close,
   &.expandable-alert .read-more,
   &.expandable-alert .read-more:hover,
   &.expandable-alert .read-more:focus {
-    color: theme-color('info')
+    color: theme-color("info");
   }
 }
 
 .alert-success {
-  border-color: theme-color('success');
+  border-color: theme-color("success");
 
   &::before {
-    content: '\E5CA';
-    background-color: theme-color-background('success');
-    color: theme-color('success')
+    color: theme-color("success");
+    content: "\E5CA";
+    background-color: theme-color-background("success");
   }
 
   .close,
   &.expandable-alert .read-more,
   &.expandable-alert .read-more:hover,
   &.expandable-alert .read-more:focus {
-    color: theme-color('success')
+    color: theme-color("success");
   }
 }
 
 .alert-warning {
-  border-color: theme-color('warning');
+  border-color: theme-color("warning");
 
   &::before {
-    content: '\E001';
-    background-color: theme-color-background('warning');
-    color: theme-color('warning')
+    color: theme-color("warning");
+    content: "\E001";
+    background-color: theme-color-background("warning");
   }
 
   .close,
   &.expandable-alert .read-more,
   &.expandable-alert .read-more:hover,
   &.expandable-alert .read-more:focus  {
-    color: theme-color('warning')
+    color: theme-color("warning");
   }
 }
 
 .alert-danger {
-  border-color: theme-color('danger');
+  border-color: theme-color("danger");
 
   &::before {
-    content: '\E001';
-    background-color: theme-color-background('danger');
-    color: theme-color('danger');
+    color: theme-color("danger");
+    content: "\E001";
+    background-color: theme-color-background("danger");
   }
 
   .close,
   &.expandable-alert .read-more,
   &.expandable-alert .read-more:hover,
   &.expandable-alert .read-more:focus  {
-    color: theme-color('danger');
+    color: theme-color("danger");
```

Comment:
Im' not sure to understand the purpose, I can see the the css doesn't change only their order..
But 1) why? 2) what is the order rule? it doesn't seem to be alphabetical 🤔
I guess all those changes were performed through automatic tools?

Reply:
Mmh the change you notified is just a character swap between double quote and single quote?

Reply:
No i meant the order of the miscellaneous css properties, but as you explained to me this is Bootsrap guideline